src/routes/ProfileRoute.tsx

```diff
@@ -1,5 +1,26 @@
 import { observer } from "mobx-react-lite"
+import { Card } from "../components/cards/Card"
+import { Avatar } from "../components/avatar/Avatar"
+import { AuthService } from "../services"
+import { Text } from "../components/typography/Text"
+import { Button } from "../components"
+import { useNavigate } from "react-router-dom"
+import { PageContainer } from "../components/other/PageContainer"
 
 export const ProfileRoute = observer(() => {
-    return <div>Profile route</div>
+    const navigate = useNavigate()
+    return (
+        <PageContainer>
+            <Card className="gap-l items-flex-start">
+                <div className="flex gap-l items-center">
+                    <Avatar source={AuthService.authenticatedUser?.photoURL || undefined} />
```

Comment:
če user nima slike, naredi da bo imel avatar iz prve črke imena in priimka. Torej John Doe bo imel sivi background avatar z JD